modules/system/controllers/updates/assets/src/components/Product.vue

```diff
@@ -146,4 +215,37 @@ export default {
 .product-footer .stars, .product-footer .github {
     margin-right: 7px;
 }
+.installing:after {
+    content: ' ';
+    display: block;
+    background-size: 50px 50px;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-image: url(/modules/system/assets/ui/images/loader-transparent.svg);
```

Comment:
Any way for this to be loaded via the ASSET_URL?

Reply:
yes there is... but this is all placeholder :innocent: